client/src/main/java/io/pravega/client/connection/impl/AppendBatchSizeTrackerImpl.java

```diff
@@ -89,8 +89,7 @@ public int getAppendBlockSize() {
         }
         double nanosPerAppend = nanosBetweenAppends.getCurrentValue();
         double appendsInMaxBatchTime = Math.max(1.0, (MAX_BATCH_TIME_MILLIS * NANOS_PER_MILLI) / nanosPerAppend);
-        double appendsInTime = Math.max(1.0, BASE_TIME_NANOS / nanosPerAppend);
```

Comment:
If the problem here is the max of 1 then why not remove that rather than deleting the value which means that BASE_TIME_NANOS cannot be configured?

Reply:
Done, I kept previous structure but changed `appendsInTime` to be effectively 0 by default, which should be the same as it was before.